plugins/exec/src/index.ts

```diff
@@ -65,8 +66,44 @@ const createEnv = (args: any[]) => ({
   ),
 });
 
+/** Wraps execSync around debugging and error handling */
+const runExecSync = (
+  command: string,
+  options: ExecSyncOptionsWithStringEncoding | undefined,
+  auto: Auto
+): string | undefined => {
+  let execResult;
+
+  try {
+    auto.logger.verbose.info(`Running command: ${command}`);
+    auto.logger.veryVerbose.info(endent`
+    Supplied Environment:
+    
+    ${Object.entries(options?.env || {})
```

Comment:
need to be careful here to not expose any tokens. in other places we sanitize the GH_TOKEN but this might be a leaky log right here

Reply:
Removed print of result values